src/Sulu/Bundle/MediaBundle/Resources/js/containers/Form/tests/fields/SingleMediaUpload.test.js

```diff
@@ -25,6 +26,20 @@ jest.mock('sulu-admin-bundle/stores/userStore', () => ({
     contentLocale: 'userContentLocale',
 }));
 
+function createMedia(data = {}): Media {
+    const exampleMedia = {
+        adminUrl: '',
+        id: 1,
+        locale: 'en',
+        mimeType: 'image/jpeg',
+        title: 'test',
+        thumbnails: {},
+        url: '',
```

Comment:
Can change it but I think its just annoying edit all tests everytime a new property is added. Would be better to have a media class in js instead just a type but that should maybe then be the case for every type which represent a entity.

Reply:
I get that, but I have just had to touch a few tests in PHP, which were written in a similar way, and they are a pain to maintain, because everytime you touch something way too much tests fail :confused:

For a similar reason we have introduced files like [`fieldTypeDefaultProps`](https://github.com/sulu/sulu/blob/9a8cf157bd8a7b1997d2d39ba4d3941aa0f14cd3/src/Sulu/Bundle/AdminBundle/Resources/js/utils/TestHelper/fieldTypeDefaultProps.js), which only contain *very* default values (almost only only empty objects and undefined). This way you don't have to write everything out, but still might save one or the other keystroke. And the tests practically don't share any data.